Docker_files/ocsci_container/scripts/run-managed-service.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+PWD=$(pwd)
+source Docker_files/ocsci_container/scripts/common.sh
+
+if [ "$AWS_PATH" == "" ]; then
```

Comment:
Do as one liner as I recommended above for setting default value if not defined

Reply:
done